net-libs/usrsctp/usrsctp-20201103.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
```

Comment:
Did you forget to use *flag-o-matic* here? I didn't get any flag errors emerging it though.

Reply:
yep. seems like someting copy/paste-related.